### PR TITLE
Save compo

### PR DESCRIPTION
**Pull request by Misty418**

### CODE REVIEW

File: ui/File/joinFile.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	this.file = file;
+	this.savedSize =
+	this.savedType = undefined;
+	this.jqToLoad.removeClass( "fa-question" )
+		.addClass( "fa-download" );
+	if ( this.fullname !== file.name ) {
+		this.fullname = file.name;
+		this.name = this.fullname.replace( /\.[^.]+$/, "" );
+		this.jqName.text( this.name )	// Awww FIX ME PLZ :'(
+			.prepend( this.jqToLoad );
```
Comment (Mr21) [inline ui/File/joinFile.js]: Yeah its fixed :)
Remove this `prepend` plz

File: ui/File/joinFile.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+ui.File.prototype.joinFile = function( file ) {
+	this.file = file;
+	this.savedSize =
+	this.savedType = undefined;
```
Comment (Mr21) [inline ui/File/joinFile.js]: We don't have to do : `= undefined` in JS normally.
Maybe you wanted to do `delete this.savedSize; delete this.savedType;`.
But I dont know if it's really necessary

File: ui/newFile.js [hunk 3]
```diff
@@ -4,4 +4,5 @@ ui.newFile = function( obj ) {
 	var uifile = new ui.File( obj );
 	ui.files.push( uifile );
 	ui.jqFilelist.append( uifile.jqFile );
+	return uifile;
```
Comment (Mr21) [inline ui/newFile.js]: But can you please remove this line too?

File: gs/samples/Sample/inTrack.js [hunk 4]
```diff
@@ -5,7 +5,10 @@ gs.Sample.prototype.inTrack = function( trackId ) {
 	if ( track !== this.track ) {
 		this.wsample.disconnect();
 		this.wsample.connect( track.wfilters );
+		if ( this.track )
```
Comment (Mr21) [inline gs/samples/Sample/inTrack.js]: {}

File: gs/reset.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+	} );
+	gs.samples.forEach( function( sample ) {
+		gs.sampleSelect( sample, true );
+	} );
+	gs.samplesDelete();
+	gs.files.forEach( function( file ) {
+		file.jqFile.remove();
+	} );
+	gs.files = [];
+};
```
Comment (Mr21) [inline gs/reset.js]: Could you please remove this file, sorry for that merge...

File: ui/File/joinFile.js (hunk 2 above)
Comment (Misty418) [inline ui/File/joinFile.js]: ok done

File: gs/samples/Sample/inTrack.js (hunk 4 above)
Comment (Misty418) [inline gs/samples/Sample/inTrack.js]: done

File: gs/reset.js (hunk 5 above)
Comment (Misty418) [inline gs/reset.js]: done

**Comment by Mr21** [comment]
:+1: :tada:

**Pull request closed without merging**